frontend/src/pages/Settings/Languages/index.tsx

```diff
@@ -121,12 +122,18 @@ const SettingsLanguagesView: FunctionComponent = () => {
           Sonarr (or a Movie from Radarr) to find a matching Bazarr language
           profile tag. It will use as the language profile the FIRST tag from
           Sonarr/Radarr that matches the tag of a Bazarr language profile
-          EXACTLY. If mutiple tags match, there is no guarantee as to which one
+          EXACTLY. If multiple tags match, there is no guarantee as to which one
           will be used, so choose your tag names carefully. Also, if you update
           the tag names in Sonarr/Radarr, Bazarr will detect this and repeat the
           matching process for the affected shows. However, if a show's only
           matching tag is removed from Sonarr/Radarr, Bazarr will NOT remove the
-          show's existing language profile, but keep it, as is.
+          show's existing language profile for that reason. But if you wish to
+          have language profiles removed automatically by tag value, simply
+          enter a list of one or more tags in the <b>Remove Profile Tags</b>
```

Comment:
Can you try to avoid using the b tag directly and replace with 

```
<Text fw={900}>Remove Profile Tags</Text>
```

instead?

Reply:
I'm not understanding you.
1) Why can't I use the Bold tag?  (it seems to work)
2) The text I have is inside a Message tag, not a Text one. I don't understand how to use the fw={900} in there?

Reply:
1. It is a best practice in React to have separation of concerns. <b> is an HTML tag which mixes it.

2. The Message Tag is a Text, just look inside the component Message.

Reply:
But there are other <b> that i will be working to remove, you can leave as it is, i will remove later.

Reply:
Thanks, please do so.
I just don't understand how to use a font weight inside of the Text.

Reply:
Like this https://github.com/morpheus65535/bazarr/pull/2639/files but i can apply once you merge this one.

Reply:
I think I just did that.
Please re-review.